app/src/main/res/values-pl/strings.xml

```diff
@@ -330,30 +603,184 @@
     <string name="settings__quickpay__settings__label">Próg QuickPay</string>
     <string name="settings__quickpay__settings__note">* Bitkit QuickPay obsługuje wyłącznie płatności z Twojego salda wydatków.</string>
     <string name="settings__security__title">Bezpieczeństwo i prywatność</string>
+    <string name="settings__security__swipe_balance_to_hide">Przesuń saldo, aby ukryć</string>
+    <string name="settings__security__hide_balance_on_open">Ukryj saldo po otwarciu</string>
     <string name="settings__security__clipboard">Odczytaj schowek dla wygody użytkowania</string>
+    <string name="settings__security__warn_100">Ostrzegaj przy transakcji powyżej 100 USD</string>
     <string name="settings__security__pin">Kod PIN</string>
     <string name="settings__security__pin_change">Zmień kod PIN</string>
     <string name="settings__security__pin_launch">Wymagaj kodu PIN podczas uruchamiania</string>
     <string name="settings__security__pin_idle">Wymagaj PIN-u po bezczynności</string>
     <string name="settings__security__pin_payments">Wymagaj PIN-u do płatności</string>
+    <string name="settings__security__pin_enabled">Włączone</string>
+    <string name="settings__security__pin_disabled">Wyłączone</string>
+    <string name="settings__security__use_bio">Użyj {biometryTypeName} wzamian</string>
     <string name="settings__security__footer">Po włączeniu tej opcji możesz używać {biometryTypeName} zamiast kodu PIN do odblokowania portfela lub wysyłania płatności.</string>
+    <string name="settings__backup__title">Utwórz kopię zapasową lub przywróć</string>
+    <string name="settings__backup__wallet">Utwórz kopię zapasową swojego portfela</string>
+    <string name="settings__backup__export">Eksport danych portfela do telefonu</string>
+    <string name="settings__backup__reset">Resetowanie i przywracanie portfela</string>
+    <string name="settings__backup__failed_title">Błąd tworzenia kopii zapasowej danych</string>
+    <string name="settings__backup__failed_message">Bitkit nie wykonał kopii zapasowej danych portfela. Ponowna próba za {interval, plural, one {# minute} other {# minutes}}.</string>
```

Comment:
This is not properly translated in Polish for numeric values:
```
{interval, plural, one {# minute} other {# minutes}}
```

- `minute` in `{# minute}` should be the singular form of minute in PL
- `minutes` in `{# minutes}` should be the singular form of minute in PL

cc @piotr-iohk